src/components/InstallTabs/index.tsx

```diff
@@ -18,27 +18,29 @@ const InstallTabs = (): JSX.Element => {
         <Tab>nvm (Linux)</Tab>
         <Tab>nvm (macOS)</Tab>
         <Tab>Chocolatey (Windows)</Tab>
-        <Tab>apt-get (Debian)</Tab>
       </TabList>
 
       <TabPanel>
         <div>
-          <code className="install__text">
-            <div className="install__text__line">
-              <span className="install__text__no-select">$</span>apk add -U curl
-              bash ca-certificates openssl ncurses coreutils python2 make gcc
-              g++ libgcc linux-headers grep util-linux binutils findutils
-            </div>
-            <div className="install__text__line">
-              <span className="install__text__no-select">$</span>curl -o-
-              https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh |
-              bash
-            </div>
-            <div className="install__text__line">
-              <span className="install__text__no-select">$</span>
-              nvm install --lts
-            </div>
-          </code>
+          <ShellBox
+            textToCopy="apk add -U curl
+bash ca-certificates openssl ncurses coreutils python2 make gcc
+g++ libgcc linux-headers grep util-linux binutils findutils"
+          >
+            <span className="install__text__no-select">$</span>
+            <span className="install__text__command">apk add -U</span> curl bash
+            ca-certificates openssl ncurses coreutils python2 make gcc g++
```

Comment:
is there a reason the default instructions here are using apk, which is alpine linux specific?

I also think that it would likely be a good idea to have MacOS first as I believe it is the most downloaded version of our distro

Reply:
I don't remember why I made it alpine specific. Pretty sure it was copy pasta from someplace 🤦  

I was wondering what the most popular OS for Node was. I'll make this change now.